README.md

```diff
@@ -15,251 +15,191 @@
 
-> See [JDABuilder](https://docs.jda.wiki/net/dv8tion/jda/api/JDABuilder.html)
-  and [DefaultShardManagerBuilder](https://docs.jda.wiki/net/dv8tion/jda/api/sharding/DefaultShardManagerBuilder.html)
+Every bot implemented by JDA starts out using the [JDABuilder][JDABuilder] or [DefaultShardManagerBuilder][DefaultShardManagerBuilder]. Both builders provide a set of default presets for cache usage and events it wants to receive:
 
-You can configure the memory usage by changing enabled `CacheFlags` on the `JDABuilder`.
-Additionally, you can change the handling of the member/user cache by disabling **intents** or changing the **member cache policy**.
-To learn more about intents and member loading/caching, read the [Gateway Intents Guide](https://jda.wiki/using-jda/gateway-intents-and-member-cache-policy/).
+- `createDefault` - Enables cache for users who are active in voice channels and all cache flags
+- `createLight` - Disables all user cache and cache flags
+- `create` - Enables member chunking, caches all users, and enables all cache flags
 
-```java
-public void configureMemoryUsage(JDABuilder builder) {
-    // Disable cache for member activities (streaming/games/spotify)
-    builder.disableCache(CacheFlag.ACTIVITY);
+We recommend reading the guide on [caching and intents](https://jda.wiki/using-jda/gateway-intents-and-member-cache-policy/) to get a feel for configuring your bot properly. Here are some possible use-cases:
 
-    // Only cache members who are either in a voice channel or owner of the guild
-    builder.setMemberCachePolicy(MemberCachePolicy.VOICE.or(MemberCachePolicy.OWNER));
+### Example: Message Logging
 
-    // Disable member chunking on startup
-    builder.setChunkingFilter(ChunkingFilter.NONE);
+Simply logging messages to the console. Making use of [JDABuilder][JDABuilder], the intended entry point for smaller bots that don't intend to grow to thousands of guilds.
 
-    // Disable presence updates and typing events
-    builder.disableIntents(GatewayIntent.GUILD_PRESENCE, GatewayIntent.GUILD_MESSAGE_TYPING);
+Starting your bot and attaching an event listener, using the right [intents][GatewayIntent]:
 
-    // Consider guilds with more than 50 members as "large". 
-    // Large guilds will only provide online members in their setup and thus reduce bandwidth if chunking is disabled.
-    builder.setLargeThreshold(50);
+```java
+public static void main(String[] args) {
+  JDABuilder.createLight(token, EnumSet.of(GatewayIntent.GUILD_MESSAGES, GatewayIntent.MESSAGE_CONTENT))
+      .addEventListener(new MessageReceiveListener())
+      .build();
 }
 ```
 
-### Listening for Events
+> [!CAUTION]
+> This also makes use of a **privileged** intent, which you have to explicitly enable in your application dashboard when you create your bot.
```

Comment:
We should say which intent is privileged and link to the discord docs explaining it.

Reply:
Is there a WARN instead of CAUTION we could use here? or perhaps NOTE. The CAUTION feels a big strong. We may also want to move this callout to be _just after_ `### Example: Message Logging` so that it informs you immediately about the concept of privileged intents and that the following example will use them

"The following example makes use of a privileged intent, MESSAGE_CONTENT, to read the content on messages. You can learn more about privileged intents [here]"